### ISSUE TITLE
Regarding File Load

### ISSUE THREAD

**Issue by toxifiedmatrix** [webtorrent/webtorrent#1881]
Hello, when I am posting a torrent file, in the input box to define the `torrentId` variable, the video isn't loading, please have a look, https://codepen.io/ToxifiedM/pen/xxZrYNG

**Comment by alxhotel** [comment]
Your example is working for me. Try using this magnet link (Sintel video):

```
magnet:?xt=urn:btih:08ada5a7a6183aae1e09d831df6748d566095a10&dn=Sintel&tr=wss%3A%2F%2Ftracker.btorrent.xyz&tr=wss%3A%2F%2Ftracker.fastcast.nz&tr=wss%3A%2F%2Ftracker.openwebtorrent.com&ws=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2F&xs=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2Fsintel.torrent
```

Also note that you are limiting the length of the input to `250` chars in the input `torrentId`. That might cause problems if your webtorrent trackers and webseeds are at the very end of the string.

**Comment by toxifiedmatrix** [comment]
Hey, @alxhotel only this example works, as when I am posting `https://yts.mx/torrent/download/952184304359E46BC91F3921B8BD17E1357CD4CB` this link, it just loads and the video never shows up, while on instant.io, it is streaming perfectly, and I have removed the max-length limit for the characters, what else can be the problem?

**Comment by toxifiedmatrix** [comment]
And in case of `magnet:?xt=urn:btih:1d5bbcbd1c87bcc1e7b48604b46a81b615c4fbbc&dn=Irresistible.2020.WEBRip.x264-ION10&tr=http%3A%2F%2Ftracker.trackerfix.com%3A80%2Fannounce&tr=udp%3A%2F%2F9.rarbg.me%3A2800&tr=udp%3A%2F%2F9.rarbg.to%3A2780` such links, the video part, the black patch doesn't even shows up, its just plain white screen, and this same link loads on the instant.io, very well. If it uses the same script which I am using, then why its not loading?

**Comment by alxhotel** [comment]
Hi @toxifiedmatrix,

Thanks for the feedback.

TL;DR Your torrent and magnet link does not seem to have any **webtorrent** peers. They only have **bittorrent** peers. (check out [Webtorrent FAQ](https://webtorrent.io/faq))

The reason why you see a video (with a black screen) appear when you use the torrent file, is because the torrent file has **metadata** about the torrent (like what files are inside the torrent). Thus, it triggers your `file.appendTo('body')` but nothing happens because there are no webtorrent peers. This doesn't happen with the magnet link, because it doesn't even have the metadata of the torrent.

Also, please **do not** post copyrighted material.

**Comment by toxifiedmatrix** [comment]
@alxhotel Okay, I see, if it doesn't have any webtorrent peers, so howcome it loads on the instant.io? It uses the same function right? Or the script is modified?

**Comment by alxhotel** [comment]
If it works in instant.io and not in the codepen, it's most probably because instant.io includes some default webtorrent trackers.
